src/commands/project/pipeline.ts

```diff
@@ -143,7 +143,7 @@ export const fetchValidateValues = (
     orgName: opts.orgName || "",
```

Comment:
This whole logic section is kinda confusing. Why are we still loading from `opts` from here. Shouldn't all of these be loaded from the `values` object instead? 

This was introduced in: https://github.com/CatalystCode/spk/pull/479/files

@edaena @dennisseah does  this make sense for us to fix is all here? Does similar logic for other ConfigValues interfaces exist now too?

Reply:
@mtarng do you know other places that this problem too?

Reply:
I'm not sure, which is why I asked the question @dennisseah . Have we refactored any other commands to move arguments from `CommandOptions` to `ConfigValues`>

Reply:
I will double check